app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> int:
+    cache_result = dict()
```

Comment:
```suggestion
    cache_result = {}
```
It's more common practice to create dict using literal. `{}` syntax works faster. you can check it on your own :)